src/main.cpp

```diff
@@ -288,7 +288,7 @@ int init_options(int argc, char**argv, struct options& options)
       }
       else
       {
-        TRC_INFO("Invalid --sas option, SAS disabled\n");
```

Comment:
In most of our components this is handled in the main function, which also encompasses the case that the --sas flag has not been supplied, so I don't think that this change is necessary. In the other components, I have added this logging to the main function so that the approach is consistent.

Reply:
Yep, I'm happy with this